draft-ietf-rtcweb-jsep.xml

```diff
@@ -1710,116 +1710,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
-
-            <t>
-            <xref target="RFC4566"></xref> is the base SDP specification
-            and MUST be implemented.</t>
-
-            <t>
-            <xref target="RFC5764"></xref> MUST be supported for
-            signaling the UDP/TLS/RTP/SAVPF
-            <xref target="RFC5764" />, TCP/DTLS/RTP/SAVPF
-            <xref target="RFC7850" />,
-            "UDP/DTLS/SCTP"
-            <xref target="I-D.ietf-mmusic-sctp-sdp" />, and
-            "TCP/DTLS/SCTP"
-            <xref target="I-D.ietf-mmusic-sctp-sdp" /> RTP profiles.</t>
-
-            <t>
-            <xref target="RFC5245"></xref> MUST be implemented for
-            signaling the ICE credentials and candidate lines
-            corresponding to each media stream. The ICE implementation
-            MUST be a Full implementation, not a Lite
-            implementation.</t>
-
-            <t>
-            <xref target="RFC5763"></xref> MUST be implemented to signal
-            DTLS certificate fingerprints.</t>
-
-            <t>
-            <xref target="RFC5888"></xref> MUST be
-            implemented for signaling grouping information, and MUST be
-            used to identify m= lines via the a=mid attribute.</t>
-
-            <t>
-            <xref target="I-D.ietf-mmusic-msid"></xref> MUST be
-            supported, in order to signal associations between RTP
-            objects and W3C MediaStreams and MediaStreamTracks in a
-            standard way.</t>
-
-            <t>The bundle mechanism in
-            <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation">
-            </xref> MUST be supported to signal the ability to multiplex
-            RTP streams on a single UDP port, in order to avoid
-            excessive use of port number resources.</t>
-
-            <t>The SDP attributes of "sendonly", "recvonly",
-            "inactive", and "sendrecv" from
-            <xref target="RFC4566"></xref> MUST be implemented to signal
-            information about media direction.</t>
-
-            <t>
-            <xref target="RFC5576"></xref> MUST be implemented to signal
-            RTP SSRC values and grouping semantics.</t>
-
-            <t>
-            <xref target="RFC4585"></xref> MUST be implemented to signal
-            RTCP based feedback.</t>
-
-            <t>
-            <xref target="RFC5761"></xref> MUST be implemented to signal
-            multiplexing of RTP and RTCP.</t>
-
-            <t>
-            <xref target="RFC5506"></xref> MUST be implemented to signal
-            reduced-size RTCP messages.</t>
-
-            <t>
-            <xref target="RFC4588"></xref> MUST be implemented to signal
-            RTX payload type associations.</t>
-
-            <t>
-            <xref target="RFC3556"></xref> MUST be supported for control
-            of RTCP bandwidth limits.</t>
-
-          </list></t>
+        <section title="Implementation Requirements" anchor="sec.implementation-requirements">
```

Comment:
Is this section still useful at this point? Should it be combined into the section below instead?